Jessy/email_ids_challenge.py

```diff
@@ -0,0 +1,52 @@
+# You run a startup media company called Ripple Media
+# It's typical when you hire a new employee in your company, to setup an email id for them
+
+print('Question 1')
+employee_name = 'Ash Rahman'
+
+# You have decided the format of the email should be: Ash Rahman -> ash.rahman@ripplemedia.com
+# Let's write some code that converts a name into an email id that matches this format
+# 1.1 TODO: Let's save the lowercase version of the employee_name in a new variable 'lower_name'
+lower_name= employee_name.lower()
+print(lower_name)
+# I simply used the conversion function to lower all the letters.
+# 1.2 TODO: We want to separate the first name and last name and save it in a variable 'names_list'
+name_list = lower_name.split()
+print(name_list)
+
+# 1.3 TODO: We want to join the first name and last name with a '.' and save it in a variable called 'joined_names'
+# joined_names
+# This was tough, had to look for videos online. I don't remember viewing .split, .join videos on Canvas.....But I love it1!!
+joined_names='.'.join(name_list)
+print(joined_names)
+# 1.4 TODO: We want to add '@ripplemedia.com' to the end of the string inside joined_names and save it in a variable 'email'
+# I had a hard time with this question
+email=joined_names+"@ripplemedia.com"
+print(email)
+print()
+
+print('Question 2')
+
+# Congratulations! Your team is expanding. Below is a list of their names:
+names = ['Max Bartlett', 'Angelita Norris', 'Stewart Mueller', 'Dominique Henry', 'Carmela Gross', 'Bettie Mcmillan', 'Sara Ellison', 'Ira Anthony', 'Pauline Riley', 'Ben Weber',
+         'Joanne Mcknight', 'Loren Gould', 'Jamar Singh', 'Amanda Vance', 'Tyrell Andrade', 'Jana Clements', 'Eddy Mcbride', 'Marsha Meyer', 'Elbert Shannon', 'Alyce Hull']
+
+emails = []
+for i in names:
```

Comment:
this added the " @ripplemedia.com " string to the list of names without changing their format like you did in question 1, so each person looks like this `'Max Bartlett@ripplemedia.com', 'Angelita Norris@ripplemedia.com',` etc

We want to convert all their names into the same format from Question 1